src/haz3lcore/statics/Translation.re

```diff
@@ -0,0 +1,402 @@
+let rec get_roc_exp = (t: TermBase.UExp.t, m: Statics.map): TermRoc.UExp.t =>
+  switch (t.term) {
+  | Invalid(_) => String("Invalid Not Implemented")
+  | EmptyHole => String("EmptyHole Not Implemented")
+  | MultiHole(_) => String("MultiHole Not Implemented")
+  | Tag(_) => String("Tag Not Implemented")
+  | Triv => Record([])
+  | Bool(b) => Bool(b)
+  | Int(n) => Int(n)
+  | Float(f) => Float(f)
+  | String(s) => String(s)
+  | ListLit(l) => ListLit(get_roc_exp_list(l, m))
+  | Fun(p, t) => Fun(get_roc_pat(p, m), get_roc_exp(t, m))
+  | Tuple(l) => Record(get_roc_exp_list(l, m))
+  | Var(token) => Var(get_name(m, token))
+  | Let(pat, def, body) =>
+    let indent_flag =
+      switch (def.term) {
+      | Let(_) => true
+      | _ => false
```

Comment:
instead of using a wildcard pattern here, it is good practice to list all the possibilities so that if something new is added to the language that requires indentation, the exhaustiveness checker forces consideration of what should happen here

Reply:
also, does a match not require indentation? or is it only nested lets?